app/scripts/directives/nav.js

```diff
@@ -1,25 +1,40 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .directive('sidebar', function(HawtioNav) {
+  .directive('sidebar', function($location, Constants) {
+    var itemMatchesPath = function(item, path) {
+      return (item.href === path) || _.some(item.prefixes, function(prefix) {
```

Comment:
Do we need to worry about trailing slashes comparing href? Like `/overview` and `/overview/`?